bot/src/application/chart/ChartService.ts

```
@@ -111,18 +120,17 @@ export class ChartService {
             // Get characters from the group
             const characterRepository =
               this.repositoryManager.getCharacterRepository();
-            const group = await characterRepository.getGroupWithCharacters(
-              groupId
-            );
+            const groups = await characterRepository.getAllCharacterGroups();
+            const group = groups.find((g) => g.id === groupId);
 
             if (group && group.characters) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)